### DISCUSSION TITLE
How to remove tabs in the top bar ?

### DISCUSSION THREAD

**Discussion by pclucas14** [alshedivat/al-folio#2031] [Q&A] [answered]
Hi, 

Thanks for the great design :). I want to remove the project / repositories / cv / teaching tabs at the top. How does one do that ?

**Comment by nWestie** [comment, marked as answer]
you can remove the tab for a specific page by setting 'nav' to false in the that pages markdown front-matter.
Ex in `projects.md`, the front matter is
```
---
layout: page
title: projects
permalink: /projects/
description: A growing collection of your cool projects.
nav: true
nav_order: 3
display_categories: [work, fun]
horizontal: false
---
```
change it to `nav: false`

It you could also probably just delete the file for that tab if you don't plan on using it.

**Comment by pclucas14** [comment]
Thank you, that works :)

Cheers

**Comment by pclucas14** [comment]
Hi, 

how does one remove the `submenus` tab ?

**Reply by george-gca** [reply]
In the exact same way as above. It's file is [_pages/dropdown.md](https://github.com/alshedivat/al-folio/blob/master/_pages/dropdown.md).

**Reply by pclucas14** [reply]
you da real MVP, thank you for the great and quick answers.

**Comment by richik-p** [comment]
@george-gca How to remove the people tab? I can find it in the list of files under `./_pages`

**Reply by nWestie** [reply]
the file is `./_pages/profiles.md` ([here](https://github.com/alshedivat/al-folio/blob/master/_pages/profiles.md))

**Reply by george-gca** [reply]
The pages are displayed with the name set in the frontmatter as `title`.

**Comment by lyygua** [comment]
Then how to remove news tab? I cannot find the `nav` in _pages/news.md.

**Reply by george-gca** [reply]
News is not a tab, it just appears inside the `about` tab. To remove it simply set `news: false` in [_pages/about.md](https://github.com/alshedivat/al-folio/blob/master/_pages/about.md) front matter.
